VERSION

```diff
@@ -1 +1 @@
-0.4.302
+0.4.303
```

Comment:
No need to bother with the version in future, I'll always make sure to take care of that.